bin/ream/src/cli/mod.rs

```diff
@@ -86,6 +86,15 @@ pub struct NodeConfig {
         help = "One or more comma-delimited base64-encoded ENR's of peers to initially connect to. Use 'default' to use the default bootnodes for the network. Use 'none' to disable bootnodes."
     )]
     pub bootnodes: Bootnodes,
+
+    #[arg(
+        long = "checkpoint-sync-url",
+        help = "Trusted RPC URL to initiate Checkpoint Sync."
+    )]
+    pub checkpoint_sync_url: Option<String>,
```

Comment:
```suggestion
    pub checkpoint_sync_url: Option<Url>,
```

we should use Url instead of a string